source/hassio/installation.markdown

```diff
@@ -64,6 +64,20 @@ curl -sL https://raw.githubusercontent.com/home-assistant/hassio-build/master/in
 When you use this installation method, the core SSH add-on may not function correctly. If that happens, use the community SSH add-on. Some of the documentation might not work for your installation either.
 </p>
 
+<p class='note'>
+  This is the list of packages that Hass.io would like to have / require:
+    apparmor-utils
+    apt-transport-https
+    avahi-daemon
+    ca-certificates
+    curl
+    dbus
+    jq
+    network-manager
+    socat
+    software-properties-common
+</p>
```

Comment:
I was thinking about adding them when I wrote the blog post. I didn't do it because it would assume that you want to use Ubuntu/Debian. I guess that now nobody will try it with Fedora/Alpine Linux anymore because Apparmor is sued..